modules/ROOT/pages/using-the-api-console.adoc

```diff
@@ -0,0 +1,34 @@
+= Using the API Console 
```

Comment:
We need to move away from focusing on the UI and focus on the user tasks. What would they use the API Console for? What are the tasks they can accomplish?
View and Test APIs